bindings/pyroot/pythonizations/test/ttreereader_iterable.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+import ROOT
+
+
+class TTreeReaderIterable(unittest.TestCase):
+    """
+    Test for the pythonization that makes instances of TTreeReader
+    iterable in Python.
+    """
+
+    # Tests
+    def test_iterable(self):
+        # Create test tree
+        filename = "ttreereader_iterable.root"
+        rdf = ROOT.RDataFrame(10).Define("x", "int(rdfentry_)").Snapshot("t", filename)
```

Comment:
would be cleaner to work on a in-memory tree, so the test does not create extra files (and runs faster)